test/testutil/cid_generator.hpp

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CPP_FILECOIN_CID_GENERATOR_HPP
+#define CPP_FILECOIN_CID_GENERATOR_HPP
+
+#include <libp2p/crypto/random_generator/boost_generator.hpp>
+#include <libp2p/multi/content_identifier.hpp>
+
+namespace testutils {
+
+  using libp2p::crypto::random::BoostRandomGenerator;
+  using libp2p::crypto::random::CSPRNG;
+  using libp2p::multi::ContentIdentifier;
+  using libp2p::multi::HashType;
+  using libp2p::multi::MulticodecType;
+  using libp2p::multi::Multihash;
+
+  /**
+   * @brief Generate random Content Identifier (CID)
+   */
+  class CidGenerator {
+   public:
+    /**
+     * Make random Content Identifier
+     * @return CID
+     */
+    ContentIdentifier makeCid() {
```

Comment:
```suggestion
    ContentIdentifier makeRandomCid() {
```
Consistency proposal. You can ignore.